source/code-snippets/usage-examples/findOne.ts

```diff
@@ -0,0 +1,40 @@
+import { FindOptions, MongoClient } from "mongodb";
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?writeConcern=majority";
+
+const client = new MongoClient(uri);
+
+type imdb = {
+  rating: number;
+  votes: number;
+  id: number;
+};
+
+interface MovieSchema extends Document {
+  title: string;
+  imdb: imdb;
+}
+
+async function run(): Promise<void> {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const movies = database.collection<MovieSchema>("movies");
```

Comment:
```suggestion
    // Specifying a Schema is always optional, but it enables type hinting on finds and inserts
    const movies = database.collection<MovieSchema>("movies");
```

I think its good that we'll be showing TS examples next to JS but it should be made clear that none of the TS features are requirements for TS users, the JS examples are valid TS examples.

Reply:
Good point! I'll try and communicate that TS features are optional in future TS rewrites of usage examples.